cmd/liked/cmd/bech32.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+// Based on: https://github.com/osmosis-labs/osmosis/blob/v7.1.0/cmd/osmosisd/cmd/bech32.go
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+)
+
+var (
+	flagToPrefix = "prefix"
+)
+
+// get cmd to convert any bech32 address to an like prefix
+func ConvertPrefixCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "convert-prefix [bech32 address]",
+		Short: "Convert any bech32 address to the like prefix",
+		Long: `Convert any bech32 address to the like prefix
+Especially useful for converting cosmos prefix to like prefix
+Example:
+	liked convert-prefix cosmos1ey69r37gfxvxg62sh4r0ktpuc46pzjrmz29g45
```

Comment:
In osmosis i actually need to do `osmosisd debug convert-prefix`
Is ours behind `liked debug` too?

Reply:
yes, `convert-prefix` and `show-height` is now under `debug`

let's remove the long description, seems the built-in usage is good enough:

```sh
./build/liked debug convert-prefix --help
Convert any bech32 address to the like prefix

Usage:
  liked debug convert-prefix [bech32 address] [flags]

Flags:
  -h, --help            help for convert-prefix
  -p, --prefix string   Bech32 prefix to encode to (default "like")

Global Flags:
      --home string         directory for config and data (default "/Users/chihim/.liked")
      --log_format string   The logging format (json|plain) (default "plain")
      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
      --trace               print out full stack trace on errors
```